dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -284,6 +294,7 @@ function pilot_death( p, attacker )
       if top_hunter == nil or player_hits >= top_hits then
          succeed()
          target_killed = true
+	 misn.appendLog("Pirate killed, and you did most damage!")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.